src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -155,13 +164,13 @@ class IdologyQuestions extends React.Component {
                     </View>
 
                     <FormAlertWithSubmitButton
-                        isAlertVisible={Boolean(this.state.errorMessage)}
```

Comment:
not sure what to do about this, it's used to let the user know they need to answwer a question

Reply:
We'll still keep this to show when the user does not select an answer and tries to submit.